src/org/labkey/remoteapi/puppeteer/PuppeteerSettings.java

```diff
@@ -50,7 +52,7 @@ public PuppeteerSettings(JSONObject json)
 
     public PuppeteerSettings(CommandResponse response)
     {
-        this(new JSONObject(response.getParsedData().get("data")));
+        this(new JSONObject((Map<String, Object>)response.getParsedData().get("data")));
```

Comment:
I don't think this will change anything. I don't think there's any particular reason for this to take a JSONObject. Could make the constructor just take the map, or use `opt` methods to pull the properties from the JSONObject.

Reply:
This causes the correct JSONObject `Map` constructor to be called (methods bind at compile time and we really don't want that `Object` version to be called). Agreed that JSONObject isn't really needed, but was trying for minimal changes. I expect to use `opt*` but first temp logging to get TC to show me what's really optional and what's not.